resources/init-container.sh

```diff
@@ -5,7 +5,11 @@ rm -f keyboard/index.cache
 cd keyboard
 php -d include_path=/var/www/html/_includes:. _build_cache.php
 
-echo "---- Generating CDN ---"
-cd ../cdn
-php -d include_path=/var/www/html/_includes:. cdnrefresh.php
-cd ..
+if [[ ! $1 =~ "debug" ]]; then
+  echo "---- Generating CDN ---"
+  cd ../cdn
+  php -d include_path=/var/www/html/_includes:. cdnrefresh.php
+  cd ..
+else
+  echo "Skip Generating CDN"
```

Comment:
I just realized that we should remove cdn/deploy if it exists (e.g. from a previous run). I think:

```suggestion
  echo "Skip Generating CDN and clean CDN cache"
  rm -rf ../cdn/deploy
```